dapps/assets/src/components/Assets.vue

```diff
@@ -0,0 +1,83 @@
+/*
+  Copyright (C) 2018-present evan GmbH.
+
+  This program is free software: you can redistribute it and/or modify it
+  under the terms of the GNU Affero General Public License, version 3,
+  as published by the Free Software Foundation.
+
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+  See the GNU Affero General Public License for more details.
+
+  You should have received a copy of the GNU Affero General Public License
+  along with this program. If not, see http://www.gnu.org/licenses/ or
+  write to the Free Software Foundation, Inc., 51 Franklin Street,
+  Fifth Floor, Boston, MA, 02110-1301 USA, or download the license from
+  the following URL: https://evan.network/license/
+*/
+
+<template>
+  <div class="evan theme-evan">
+    <evan-dapp-wrapper>
+      <template v-slot:content>
+        <evan-dapp-wrapper-level-2 ref="level2Wrapper">
+          <div class="sidenav">
+            <!-- Not using nav-list because it doesnt support router-link properly
+            TODO: Refactor evan-nav-list to use router-links too -->
+            <div class="evan-nav-list">
+              <div class="nav-entries">
+                <template v-for="navItem in navItems">
+                  <router-link
+                    :id="navItem.id"
+                    :key="navItem.id"
+                    :to="navItem.to"
+                    :active-class="'active'"
+                  >
+                    <i class="mr-3" :class="navItem.icon"></i>
+                    {{ navItem.label | translate }}
+                  </router-link>
+                </template>
+              </div>
+              <div class="my-auto"></div>
+              <div class="nav-entries">
+                <div class="bg-white border-top d-flex">
+                  <i class="mdi mdi-magnify text-muted p-3"></i>
+                  <input
+                    placeholder="0x123ABC..."
+                    class="border-0 p-3 w-100 form-control"
+                    style="height: 60px;"
+                    v-model="searchQuery"
+                    @keyup.enter="handleSearchEnter"
+                  />
+                </div>
+              </div>
+            </div>
+          </div>
+        </evan-dapp-wrapper-level-2>
+        <transition name="fade" mode="out-in">
+          <div class="container content">
+            <router-view></router-view>
+          </div>
+        </transition>
+      </template>
+    </evan-dapp-wrapper>
+  </div>
+</template>
+
+<script lang="ts">
+import AssetsComponent from './Assets';
+export default AssetsComponent;
+</script>
+
+<style lang="scss" scoped>
+.content {
+  max-width: 765px;
```

Comment:
can we use one of the breakpoint vars here to keep it dynamic?

Reply:
I would also suggest to put it in scss file, also it's just a few lines

Reply:
Only a few lines is rather an argument to not create an extra file for it imo.